linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="root_permissions_syslibrary_files" version="2">
+    {{{ oval_metadata("
+        Checks that system-wide library files in /lib, /lib64, /usr/lib, /usr/lib64
+        are owned by root.
+      ") }}}
+    <criteria >
+      <criterion test_ref="test_root_permissions_for_syslibrary_files" />
+    </criteria>
+  </definition>
+
+  <unix:file_test  check="all" check_existence="none_exist" comment="test if system-wide files have root permissions" id="test_root_permissions_for_syslibrary_files" version="1">
+    <unix:object object_ref="root_permissions_for_system_wide_library_files" />
+  </unix:file_test>
+
+  <unix:file_object comment="system-wide directories" id="root_permissions_for_system_wide_library_files" version="1">
+    <!-- Checks that system-wide library files in /lib, /lib64, /usr/lib, /usr/lib64
+        are owned by root. -->
+    <unix:path operation="pattern match">^\/lib[64?]|^\/usr\/lib[64?]</unix:path >    
+    <unix:filename operation="pattern match">^.*$</unix:filename>
```

Comment:
If you want to recurse directories, please use file behaviors.
You can se an example here:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/selinux/selinux_all_devicefiles_labeled/oval/shared.xml#L12

Reply:
This is the only remaining part to be fixed. This is excerpt from the OVAL specification:
Note that the default recurse_direction behavior is 'none' so even though max_depth specifies no limitation by default, the recurse_direction behavior turns recursion off.

Reply:
Hi @vojtapolasek  - I have implemented the proposed change and I have corrected the regex.